.kitchen.yml

```diff
@@ -16,7 +16,8 @@ platforms:
   - name: debian-8.4
   - name: windows
     driver:
-      box: opentable/win-2012r2-standard-amd64-nocm
+      box: mwrock/Windows2012r2
+      gui: false # This box apparently defaults to using a GUI.
```

Comment:
He reassures me that running it without gui is fine, and he often runs it this way as well. It's just a flag to determine whether or not virtualbox opens a gui, so I assume in his workflows he often wants a gui to open from running vagrant vms.